docs/create_action_scripts.md

```
@@ -9,40 +9,31 @@ There is a checklist before selecting action script as your solution:
 ## Regular action scripts
 
-There is some different types of action scripts based on their base classes, `ActionScript` class designed for regular
-action scripts and can handle a lot of tasks.
+There is some different types of action scripts based on their base classes, `ActionScript` class designed for regular action scripts and can handle a lot of tasks.
 
-To add new action script you should make sure there is an item in menus for that action script, you can use `data/main_ui.ini`
-to add new options (or menus), to understand about structure of this file you can see [here](https://text-forge.github.io/docs/data_driven_ui/#menus).
-After select your option create a script in `action_scripts/` directory with same name (but in snake_case), it this case
-we will create **Copy Path** action script that will copy file path of opened file in clipboard, so we will create
-`action_scripts/copy_path.gd` and use this script as start point:
+To add a new action script, first ensure there is a matching menu item. You can add options (or menus) in `data/main_ui.ini`; see the structure [here](https://text-forge.github.io/docs/data_driven_ui/#menus). After selecting your option, create a script in the `action_scripts/` directory with the same name in snake_case. In this example, we will create a **Copy Path** action script to copy the currently opened file path to the clipboard, so we create `action_scripts/copy_path.gd` and start with:
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)